dartagnan/src/main/java/com/dat3m/dartagnan/wmm/analysis/RelationAnalysis.java

```diff
@@ -262,6 +260,15 @@ private void run() {
                 if (delta.may.isEmpty() && delta.must.isEmpty()) {
                     continue;
                 }
+
+                // TODO: Quick fix for https://github.com/hernanponcedeleon/Dat3M/issues/523
```

Comment:
Quick fix for the old issue to make GPU unit tests pass. A proper fix is needed.